.github/workflows/release.yaml

```diff
@@ -25,6 +25,76 @@ jobs:
           version: v3.12.0
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.5.0
+        id: chart_releaser
+        uses: helm/chart-releaser-action@v1.6.0
```

Comment:
Bumped to https://github.com/helm/chart-releaser-action/releases/tag/v1.6.0 to retrieve the new `changed_charts` output.